dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -47,6 +47,17 @@ HRESULT CFSDropTarget::_CopyItems(IShellFolder * pSFFrom, UINT cidl,
     /* Build a double null terminated list of C strings from source paths */
     for (UINT i = 0; i < cidl; i++)
     {
+        SFGAOF rfg = SFGAO_FILESYSANCESTOR | SFGAO_FILESYSTEM;
+        ret = pSFFrom->GetAttributesOf(i, apidl, &rfg);
```

Comment:
This line looks a bit odd to me. Do you mean something like the following?
```suggestion
        ret = pSFFrom->GetAttributesOf(1, apidl[i], &rfg);
```